inst/private/doctest_run_docstring.m

```diff
@@ -30,7 +30,7 @@
 % extract tests from docstring
 TEST_RE = [                               % loosely based on Python 2.6 doctest.py, line 510
     '(?m)(?-s)'                          ... % options
-    '(?:^ *>> )'                         ... % ">> "
+    '(?:^ *>>>? )'                         ... % ">> " or ">>> "
```

Comment:
What about `(?:^ *>>>* )`? This would make valid prompts out of `>>`, or `>>>`, or even `>>>>>>`, without giving any special significance to Python's default `>>>`.

Reply:
We could do that, of course. Can you think of a situation, where it would be useful to have more than three `>`?